### PR TITLE
Use hooks

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

### CODE REVIEW

**Comment by thomasthiebaud** [comment]
Nice ❤️

File: package.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
     "@semantic-release/git": "^6.0.1",
     "@semantic-release/npm": "^3.4.0",
     "@semantic-release/release-notes-generator": "^6.1.1",
+    "and": "0.0.3",
```
Comment (pmrcunha) [inline package.json]: What's this?

File: README.md [hunk 2]
```diff
@@ -280,6 +281,24 @@ const date = new Date(2018, 1, 15)
 <TimeAgo date={date}/> // renders someting like "6 months ago"
 ```
 
+### useTranslations
+
+That's right, hooks! Get access to the raw translations data from the context with the `useTranslations` custom hook:
+
+```javascript
+function App() {
+  const [translations, locale] = useTranslations();
```
Comment (pmrcunha) [inline README.md]: Let's switch this to be an object

**Comment by madsgodvinjensen** [comment]
@pmrcunha I used an object instead. Also did some changes to tests - added a coverage report and import components from index.js to test the main export as well.

File: README.md [hunk 3]
```diff
@@ -280,6 +281,24 @@ const date = new Date(2018, 1, 15)
 <TimeAgo date={date}/> // renders someting like "6 months ago"
 ```
 
+### useTranslations
+
+That's right, hooks! Get access to the raw translations data from the context with the `useTranslations` custom hook:
```
Comment (pmrcunha) [inline README.md]: Is the excitement still appropriate? :P

File: package.json [hunk 4]
```diff
@@ -23,7 +23,7 @@
   "author": "Trustpilot",
   "peerDependencies": {
     "prop-types": "15.x",
-    "react": "^16.3.2"
+    "react": "16.8.x"
```
Comment (pmrcunha) [inline package.json]: Why not support 16.x ?

File: package.json (hunk 1 above)
Comment (pmrcunha) [reply package.json]: So, what's this again?

File: package.json (hunk 4 above)
Comment (madsgodvinjensen) [reply package.json]: It has to be above 16.8 for hooks. Maybe we should do `^16.8`

**Pull request merged**